lib/l10n/intl_it.arb

```
@@ -399,5 +399,42 @@
   "share": "Condividi",
   "failedToShareInvoice": "Errore nel condividere la fattura. Per favore prova a copiarla invece.",
   "openWallet": "APRI PORTAFOGLIO",
-  "done": "FATTO"
+  "done": "FATTO",
```

RuboCop (1.75.5)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)